### PR TITLE
TD works without CUBLAS library

### PR DESCRIPTION
**Pull request by gonzalodm**
en TD.f90 la linea rho1=matmul(x,rho) generaba error ya que la maztriz x y rho no son del mismo tamaño, lo unico que hize es dar las dimensiones correctas a x

### CODE REVIEW

File: lioamber/TD.f90 [hunk 1]
```diff
@@ -112,7 +112,7 @@ SUBROUTINE TD()
        ALLOCATE(factorial(NBCH))
 !!------------------------------------!!
 ! Mulliken
-       ipop=1
+       ipop=0
```
Comment (ramirezfranciscof) [inline lioamber/TD.f90]: Is this change necessary? Is it relevant for this PR?

File: lioamber/TD.f90 [hunk 2]
```diff
@@ -333,15 +331,14 @@ SUBROUTINE TD()
        dovv=.true.
        if (dovv.eqv..true.) then
         if (.not.allocated(orb_group)) then
-          allocate(orb_group(M))
+          allocate(orb_group(M),group(M)) ! agrege que allocate la variable group
```
Comment (ramirezfranciscof) [inline lioamber/TD.f90]: Are you doing this because you detected that `group` was not allocated? That variable should contain information. I am fearing that somehow an unpolished version of the atom potential implementation got inserted here. I really don't like the `dovv=.true.` above; did you try deleting that instead of allocating `group`?

**Comment by gonzalodm** [comment]
Perdon estos cambios no deberian estar aqui. Pero ipop=1 hacia que el
codigo se rompa, ya hable con charly y me dijo que lo iva a ver.
Con respecto a group, efectivamente no estaba allocateada

2017-10-09 18:24 GMT-03:00 ramirezfranciscof <notifications@github.com>:
